src/electron/platform/android/scan-results.ts

```diff
@@ -25,9 +25,25 @@ export interface BoundingRectangle {
     top: number;
 }
 
+export interface AxeDeviceInfo {
+    dpi: number;
+    name: string;
+    osVersion: string;
+    screenHeight: number;
+    screenWidth: number;
+}
+
 export class ScanResults {
     constructor(readonly rawData: any) {}
 
+    public get axeDevice(): AxeDeviceInfo {
```

Comment:
there are missing tests for this new method.

Reply:
added